api/classes/class-lightweight-api.php

```diff
@@ -231,26 +233,19 @@ public function get_transient( $name ) {
 		global $wpdb;
 		$name = '_transient_' . $name;
 
-		$empty_transients = wp_cache_get( 'empty_transients', 'newspack-popups' );
-
-		if ( ! is_array( $empty_transients ) ) {
-			$empty_transients = array();
-		}
-
-		if ( isset( $empty_transients[ $name ] ) ) {
+		$value = wp_cache_get( $name, 'newspack-popups' );
+		if ( -1 === $value ) {
```

Comment:
This is wonderful ! , exactly what we needed here IMHO.

Reply:
Great suggestion!